### ISSUE TITLE
pdf-reader blocking links to PDF URLs

### ISSUE THREAD

**Issue by mralitsch** [Emano-Waldeck/pdf-reader#3]
Installed pdf-reader on Chrome yesterday and I'm constantly getting the following error when clicking PDF documents:

**ieepebpjnkhaiioojkepfniodjmjjihl is blocked**
Requests to the server have been blocked by an extension.
Try disabling your extensions.
ERR_BLOCKED_BY_CLIENT

Disabling pdf-reader eliminates the error message and allows access to the PDF documents. 

Chrome Version 85.0.4183.83 (Official Build) beta (64-bit)
pdf-reader version 0.1.4

**Comment by keyansheng** [comment]
Duplicate of #2
